docs/tutorials/introduction.rst

```diff
@@ -151,35 +146,23 @@ self-explanatory by looking at our example.
      {name = 'Bowsette Koopa', email = 'bowsette@example.com'},
    ]
 
-
-.. admonition:: Adding dependencies
-   :class: seealso
-
-   If you need to add dependencies to the project metadata, you can check our
-   :ref:`how-to-guides-add-dependencies` guide.
-
-
 .. admonition:: Declaring project metadata
    :class: seealso
 
-   Our example doesn't make use of all the fields available, so we recommend you
-   check out the `PyPA documentation on project metadata`_.
-
-
-.. admonition:: Writing TOML
-   :class: seealso
-
-   If you are not familiar with the TOML configuration language or need some
-   help, be sure to check out `toml.io <https://toml.io>`_.
+   Our example doesn't make use of all the fields available in the ``[project]``
+   section. Check out the `PyPA documentation on project metadata`_ for more
+   examples and details.
 
 
 Testing the project
 -------------------
 
-Now we should have a valid Python project, let's test it.
-
-We will install it with pip_
+Now we should have a valid Python project, so let's test it. Before continuing,
+ensure you have committed the three files we created above to your Git
+repository - ``meson-python`` will only take into account the files that Git
+knows about.
```

Comment:
I'm not sure mentioning here that only committed files are taken into account to build an sdist is relevant. AFAIK, `pip` does not build wheels via an sdist, this is the default behavior of `build` but not of `pip`. I would move this sentence further down where creating sdists and wheels is mentioned.

Reply:
Good point - moved down.